examples/time.rs

```diff
@@ -1,23 +1,19 @@
-use extrasafe::{SafetyContext, builtins};
-use builtins::{Time, SystemIO};
+use builtins::{SystemIO, Time};
+use extrasafe::{builtins, SafetyContext};
 
 fn main() {
     SafetyContext::new()
-        .enable(
-            SystemIO::nothing()
-                .allow_stdout()
-        ).unwrap()
-
+        .enable(SystemIO::nothing().allow_stdout())
+        .unwrap()
         // On most systems this won't have an effect because glibc and musl both use vDSOs that
         // compute time directly via rdtsc rather than calling the syscalls directly.
-        .enable(
-            Time::nothing()
-                .allow_gettime()
-        ).unwrap()
-        .apply_to_current_thread().unwrap();
+        .enable(Time::nothing().allow_gettime())
```

Comment:
I think it was the stuff like this that made me not want to format. This could be resolved by putting the RuleSet instance in its own variable I guess.

Reply:
Also putting unwrap() at the same indentation as the "real" functions being called is a bit annoying as well.